.github/workflows/pull-transifex.yml

```diff
@@ -0,0 +1,43 @@
+name: transifex
+on:
+  workflow_dispatch:
```

Comment:
The pull workflow is not in a state where we can reliably run it on a schedule. There are some issues with our translations (e.g. use of literal `%`) that will cause this workflow to error.